librz/core/disasm.c

```diff
@@ -1013,6 +1013,16 @@ static void ds_build_op_str(RzDisasmState *ds, bool print_color) {
 		}
 		return;
 	}
+	// if the <invalid> is referred as data, show bytes
```

Comment:
I think the same could be achieved using "analysis hints." Check how `CCd` is implemented.